README.md

```diff
@@ -6,22 +6,19 @@ Rust userland library for Tock (WIP)
 
 This project is nascent and still under heavy development, but first steps:
 
-1. Ensure you have a working toolchain available:
+```bash
+./build_examples.sh
```

Comment:
Technically, this step is not required. The purpose of `build_examples.sh` is to be run by Travis CI in order to make sure that examples compile.